src/cf-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "4.0.1",
+  "version": "4.2.0",
```

Comment:
We also still have this issue where it bumps two increments in a single release.